common/src/stack/command/stack/commands/report/dhcpd/__init__.py

```diff
@@ -142,15 +144,31 @@ def writeDhcpDotConf(self):
 			data[host['host']] = []
 
 		host_devices = {}
-		for interface in self.call('list.host.interface'):
+		interfaces = self.call('list.host.interface')
+		for interface in interfaces:
 			host = interface['host']
 			mac = interface['mac']
 			ip = interface['ip']
 			device = interface['interface']
-
+			channel = interface['channel']
+
+			other_interfaces = [iface['interface'] for iface in interfaces if iface['host'] == host and iface['interface'] != device]
```

Comment:
I'd suggest splitting this line, for those among us who don't like list comprehensions with repetitive variable names